.github/workflows/AIForOrcas.Client.Web.yaml

```diff
@@ -9,6 +9,7 @@ on:
     - ModeratorFrontEnd/AIForOrcas/AIForOrcas.Client.BL/**
     - ModeratorFrontEnd/AIForOrcas/AIForOrcas.Server.BL/**
     - ModeratorFrontEnd/AIForOrcas/AIForOrcas.DTO/**
+    - .github/workflows/AIForOrcas.Client.Web.yaml
```

Comment:
This means the action runs on pull request & not on merge, which means the build & not deploy runs. Is that correct? Is there a possibility that dependabot breaks the deploy dependencies?

Reply:
Seems unlikely, but I've now added it to the `push` trigger as well, just to be safe.